frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -60,7 +84,20 @@ export function IngestionWarningsView(): JSX.Element {
                         title: 'Warning',
                         dataIndex: 'type',
                         render: function Render(_, summary: IngestionWarningSummary) {
-                            return <>{WARNING_TYPE_TO_DESCRIPTION[summary.type] || summary.type}</>
+                            const type = WARNING_TYPE_TO_DESCRIPTION[summary.type] || summary.type
+                            return (
+                                <>
+                                    {type} (
+                                    <Link
+                                        to={`https://posthog.com/docs/integrate/ingestion-warnings#${type
```

Comment:
Suggestion: let's explicitly link over interpolation for maintanence reasons.

That way it's it's easier to verify whether the links are in sync (or even exist). Right now we need to either "run the code in our head" to verify or find a way to trigger warnings (non-trivial). Also someone working on posthog.com is likely to break these links involuntarily.

Reply:
I really like interpolating here. Note that we're just interpolating things after `#` if it's not on the page you'll be just taken to the page anyway, so it will still work always. e.g. try https://posthog.com/handbook/getting-started/start-here#no-such-title.

Reply:
I really like interpolating here. Note that we're just interpolating things after `#` if it's not on the page you'll be just taken to the page anyway, so it will still work always. e.g. try https://posthog.com/handbook/getting-started/start-here#no-such-title.